### DISCUSSION TITLE
Derivata di un sensore

### DISCUSSION THREAD

**Discussion by zanac** [indomus/forum#2632] [GENERALE - Discussioni generiche]
Ciao,

Ho un sensore che calcolo usando una derivata di un altro sensore (che ha il numero di litri di acqua totali e viene aggiornato in modo impeccabile.
Tutto funziona correttamente, con la derivata ottengo il consumo di litri d'acqua istantaneo... unica pecca... non riesco dare un nome parlante perchè si lamenta che "Questa entità ("non ha un ID univoco, pertanto le sue impostazioni non possono essere gestite dall'interfaccia utente. Vedi [documentazione](https://www.home-assistant.io/faq/unique_id) per maggiori dettagli.
Peccato che non mi lascia inserire unique_id con un guid quando ho la platform derivate... avete un esempio di che sintassi dovrai usare?

```
platform: derivative
name: WaterMeterInstant                                                                                                                            
source: sensor.WaterMeter                                                                                                                          
unit_time: min
time_window: "00:00:01"
unit: 'L'
round: 3
```

ciao

**Comment by 50494554524F** [comment]
usa i file customize o la sezione customize entity nella ui e gli cambi il friendly_name

**Reply by Alessandro2103** [reply]
posso sapere dove trovo il file customize o la sezione di cui parli??? grazie

**Reply by 50494554524F** [reply]
https://www.home-assistant.io/docs/configuration/customizing-devices/

**Comment by zanac** [comment]
Fantastico, è proprio quello che cercavo di fare... grazie!
